platform/viewer/src/routes/routesUtil.js

```diff
@@ -2,111 +2,282 @@ import OHIF from '@ohif/core';
 const { urlUtil: UrlUtil } = OHIF.utils;
 
 const reload = () => window.location.reload();
+const isTemplateNameEqual = (templateNameA, templateNameB) => {
```

Comment:
These all feel odd as utils to me.
I think we want to noodle on a better name and placement for these in `@ohif/core` as something that can easily work with our appConfig schema and what React Router expects for wiring up routes.
Then consume it in our application, and document its usage.

Reply:
Let's hammer this out tomorrow together.

Reply:
ok I will changed as we discussed